eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -157,6 +158,7 @@ library testlib
         cardano-strict-containers,
         containers,
         cuddle >=0.3.0.0,
+        data-default,
```

Comment:
Why another dependency? We already have `data-default-class` below.
```suggestion
```

Reply:
Yes, sorry, I was confused between the two, I could have sworn we have "Data.Default (def)"  everywhere, but evidently it's not the case.

Reply:
I also removed it from cardano-ledger-api in an additional commit.